code/modules/ninja/suit/suit.dm

```diff
@@ -97,10 +97,10 @@ Contents:
 /obj/item/clothing/suit/space/space_ninja/proc/lock_suit(mob/living/carbon/human/H)
 	if(!istype(H))
 		return FALSE
-	if(!is_ninja(H))
+	/*if(!is_ninja(H))
```

Comment:
I mean I can try one moment

Reply:
Ok so I tried a couple of ways but cant seem to figure out a way to do it without just commenting out the section I dont really think this will impact much and if someone does get a ninja suit then good on them honestly am open to ways to do this and or alt prize though

Reply:
@ThatLing

Reply:
@ThatLing >:(